Data/scripts/source/zQuestLooper.psc

```diff
@@ -0,0 +1,88 @@
+ScriptName zQuestLooper extends Quest
+
+import Skybitica
+
+; List of all named Skyrim quests defined in Creation Kit
+FormList Property quests auto
+Formlist Property runningQuests auto
+Formlist Property pendingQuests auto
+
+bool upToDate = true
+
+Event OnInit()
+	; Script will run whenever player opens journal
+	RegisterForMenu("Journal Menu")
+EndEvent
+
+Event OnMenuOpen(string menuName)
+	GetRunningQuests()
+	ConnectQuestsToHabitica() 
+EndEvent
+
+Event OnUpdate()
+	Int i = 0
+	While i < pendingQuests.GetSize()
+		Quest pendingQuest = pendingQuests.GetAt(i) as Quest
+		Int response = GetRequestResult(pendingQuest.GetId())
+		If httpRequestSucceeded(response)
+			Debug.Notification("'" + pendingQuest.GetName() + "'" + " successfully updated on Habitica!")
+			runningQuests.RemoveAddedForm(pendingQuest)
```

Comment:
This is the cause of the aforementioned bug.
Instead, quests should only be removed from ```runningQuests``` if they are complete or stopped, not if they are running.